src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/OwnersAndOfficersDataBlockValidatorTest.java

```diff
@@ -0,0 +1,177 @@
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.ManagingOfficerIndividualDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.PresenterDto;
+import uk.gov.companieshouse.overseasentitiesapi.validation.BeneficialOwnerCorporateValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.BeneficialOwnerGovernmentOrPublicAuthorityValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.BeneficialOwnerIndividualValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.BeneficialOwnersStatementValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.ManagingOfficerCorporateValidator;
+import uk.gov.companieshouse.overseasentitiesapi.validation.ManagingOfficerIndividualValidator;
+import uk.gov.companieshouse.service.rest.err.Err;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto.BENEFICIAL_OWNERS_STATEMENT;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.OwnersAndOfficersDataBlockValidator.INCORRECTLY_ADDED_BENEFICIAL_OWNER;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.OwnersAndOfficersDataBlockValidator.INCORRECTLY_ADDED_MANAGING_OFFICER;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.OwnersAndOfficersDataBlockValidator.MISSING_BENEFICIAL_OWNER;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.OwnersAndOfficersDataBlockValidator.MISSING_MANAGING_OFFICER;
+
+@ExtendWith(MockitoExtension.class)
+class OwnersAndOfficersDataBlockValidatorTest {
+
+    private static final String LOGGING_CONTEXT = "12345";
+    @InjectMocks
+    private OwnersAndOfficersDataBlockValidator ownersAndOfficersDataBlockValidator;
+    @Mock
+    private BeneficialOwnersStatementValidator beneficialOwnersStatementValidator;
+    @Mock
+    private BeneficialOwnerIndividualValidator beneficialOwnerIndividualValidator;
+    @Mock
+    private BeneficialOwnerCorporateValidator beneficialOwnerCorporateValidator;
+    @Mock
+    private BeneficialOwnerGovernmentOrPublicAuthorityValidator beneficialOwnerGovernmentOrPublicAuthorityValidator;
+    @Mock
+    private ManagingOfficerIndividualValidator managingOfficerIndividualValidator;
+    @Mock
+    private ManagingOfficerCorporateValidator managingOfficerCorporateValidator;
+
+    private OverseasEntitySubmissionDto overseasEntitySubmissionDto;
+    private EntityDto entityDto = EntityMock.getEntityDto();
+    private PresenterDto presenterDto = PresenterMock.getPresenterDto();
+    private DueDiligenceDto dueDiligenceDto = DueDiligenceMock.getDueDiligenceDto();
```

Comment:
these dto vars can be 'final'

Reply:
Marked final

Reply:
these aren't showing as final for me, maybe missed due to the package move

Reply:
Added final